app/src/main/java/com/example/weathertimeapp/mvp/entities/Weather.kt

```diff
@@ -1,14 +1,11 @@
 package com.example.weathertimeapp.mvp.entities
 
-data class Weather(
-    var date: String,
-    var maxTemp: String,
-    var minTemp: String,
-    var image: String,
-    var description: String
-) {
+import com.example.weathertimeapp.utils.DEFAULT_ID
+import com.example.weathertimeapp.utils.DEFAULT_STRING
 
-    override fun toString(): String {
-        return "Weather(date='$date', image='$image', maxTemp='$maxTemp', minTemp='$minTemp, description='$description')"
-    }
-}
\ No newline at end of file
+data class Weather(
+    val id: Int = DEFAULT_ID,
+    val main: String = DEFAULT_STRING,
```

Comment:
same as I say before

Reply:
changed, main was the type of the weather, now is called "type"